lib/instrumentation/span.js

```diff
@@ -219,6 +219,12 @@ Span.prototype._encode = function (cb) {
       outcome: self.outcome
     }
 
+    // if a valid sample rate is set (truthy or zero), set the property
+    const sampleRate = self.sampleRate
+    if (sampleRate || sampleRate === 0) {
```

Comment:
```suggestion
    if (sampleRate !== null) {
```

Do the !== null check here as well?